demo-smart/index.html

```diff
@@ -325,7 +330,11 @@
         test-addon-config
       </button>
       <button
-        data-context='{"ownerId":"orga_3547a882-d464-4c34-8168-add4b3e0c135","addonId":"addon_7bfe6a42-4cb8-488e-96db-77c8da4627d2"}'
+        data-context='{
+        "ownerId":"orga_3547a882-d464-4c34-8168-add4b3e0c135",
+        "addonId":"addon_7bfe6a42-4cb8-488e-96db-77c8da4627d2",
+        "logsUrlPattern":"/organisations/orga_3547a882-d464-4c34-8168-add4b3e0c135/applications/:id/logs"
```

Comment:
nitpick: even if we won't be able to go to the url logs, the pattern should be on `addons` not `applications`